core/javaAction/Dockerfile.OpenJ9

```diff
@@ -0,0 +1,10 @@
+FROM adoptopenjdk/openjdk8-openj9:x86_64-ubuntu-jdk8u152-b16
```

Comment:
We should replace the current image using oracle jdk and replcae the current java8action one with with this openjdk which uses ubuntu/debian based, transparent transition

Reply:
+1

Reply:
then we can close the other PR as this takes care of it ???